aiidalab_widgets_base/computational_resources.py

```diff
@@ -229,15 +232,19 @@ def _setup_new_code(self, _=None):
                     "width": "500px",
                     "border": "1px solid gray",
                 }
-                display(
-                    ipw.HTML(
-                        """Please select the computer/code from a database to pre-fill the fields below."""
-                    ),
-                    self.comp_resources_database,
-                    self.setup_message,
-                    self.ssh_computer_setup.password_box,
-                    self.output_tab,
+                children = (
```

Comment:
I assume the `children` here need to be a list?

Reply:
Yes, variable `children` is a list. (the opening parenthesis here is just for multiple line splitting)